Split test_c10d.py to test_c10d_common.py, test_c10d_gloo.py, test_c10d_nccl.py

pbelevich opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #56442 Add Gloo TCP_TLS transport
* #56506 Split test_c10d_spawn.py to test_c10d_spawn_gloo.py,test_c10d_spawn_nccl.py
* **#56435 Split test_c10d.py to test_c10d_common.py, test_c10d_gloo.py, test_c10d_nccl.py**

Differential Revision: [D27896286](https://our.internmc.facebook.com/intern/diff/D27896286)

pbelevich committed:
Split test_c10d.py to test_c10d_common.py, test_c10d_gloo.py, test_c10d_nccl.py

[ghstack-poisoned]

pbelevich committed:
Update on "Split test_c10d.py to test_c10d_common.py, test_c10d_gloo.py, test_c10d_nccl.py"




[ghstack-poisoned]

pbelevich committed:
Update on "Split test_c10d.py to test_c10d_common.py, test_c10d_gloo.py, test_c10d_nccl.py"




[ghstack-poisoned]

pbelevich committed:
Update on "Split test_c10d.py to test_c10d_common.py, test_c10d_gloo.py, test_c10d_nccl.py"




[ghstack-poisoned]

pbelevich committed:
Update on "Split test_c10d.py to test_c10d_common.py, test_c10d_gloo.py, test_c10d_nccl.py"




[ghstack-poisoned]

pbelevich committed:
Update on "Split test_c10d.py to test_c10d_common.py, test_c10d_gloo.py, test_c10d_nccl.py"




[ghstack-poisoned]

pbelevich committed:
Update on "Split test_c10d.py to test_c10d_common.py, test_c10d_gloo.py, test_c10d_nccl.py"




[ghstack-poisoned]

SciPioneer commented:
Could you please use a command like `hg cp` when splitting `test_c10d.py` into 3 smaller files rather than creating 3 new files from scratch? This way a reviewer can clearly see which part is exact copy, and which part is pure deletion. and reviewer can mainly focus on the real differences. Thanks!

pbelevich committed:
Update on "Split test_c10d.py to test_c10d_common.py, test_c10d_gloo.py, test_c10d_nccl.py"


Differential Revision: [D27896286](https://our.internmc.facebook.com/intern/diff/D27896286)

[ghstack-poisoned]

pbelevich committed:
Update on "Split test_c10d.py to test_c10d_common.py, test_c10d_gloo.py, test_c10d_nccl.py"


Differential Revision: [D27896286](https://our.internmc.facebook.com/intern/diff/D27896286)

[ghstack-poisoned]

pbelevich committed:
Update on "Split test_c10d.py to test_c10d_common.py, test_c10d_gloo.py, test_c10d_nccl.py"


Differential Revision: [D27896286](https://our.internmc.facebook.com/intern/diff/D27896286)

[ghstack-poisoned]
